### PR TITLE
[fix](ip)fix default value for ip

### PR DESCRIPTION
**Pull request by amorynan**
### What problem does this PR solve?
before this pr: we do not support create table for IP type with default value 
like this:
```
mysql> CREATE TABLE table2
    -> (
    ->     col0 BIGINT  NOT NULL DEFAULT '1' ,
    ->     col24 IPV4  NULL DEFAULT '127.0.0.1'
    -> )
    -> DUPLICATE KEY(col0)
    -> DISTRIBUTED BY HASH(col0) BUCKETS 4
    -> PROPERTIES (
    ->     "replication_num" = "1"
    -> );
ERROR 1105 (HY000): errCode = 2, detailMessage = Unsupported type: ipv4
```
after this pr we deal with this problem
Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by amorynan** [comment]
run buildall

File: regression-test/suites/datatype_p0/ip/test_ip_basic.groovy [hunk 1]
```diff
@@ -160,4 +160,11 @@ suite("test_ip_basic") {
     qt_sql """ select * from table_ip where col0 = 1"""
     sql """ Update table_ip set col25 = 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff' where col0 = 1 """
     qt_sql """ select * from table_ip where col0 = 1"""
+
+    // test ip with default value
+    sql """ DROP TABLE IF EXISTS table_ip_default """
+    sql """ CREATE TABLE IF NOT EXISTS `table_ip_default` (`col0` bigint NOT NULL, `col4` ipv6 NULL DEFAULT "::",   `col24` ipv4 NULL DEFAULT "127.0.0.1") ENGINE=OLAP UNIQUE KEY(`col0`) DISTRIBUTED BY HASH(`col0`) BUCKETS 4 PROPERTIES ("replication_allocation" = "tag.location.default: 1") """
```
Comment (yiguolei) [inline regression-test/suites/datatype_p0/ip/test_ip_basic.groovy:166]: add more 
test:
1. show create table
2. desc table
3. create table like
 to make sure in all cases, the default value is not lost.

File: regression-test/suites/datatype_p0/ip/test_ip_basic.groovy [hunk 2]
```diff
@@ -160,4 +160,11 @@ suite("test_ip_basic") {
...
+    sql """ insert into table_ip_default values (1, "5be8:dde9:7f0b:d5a7:bd01:b3be:9c69:573b", "0.0.0.1") """
+    sql """ insert into table_ip_default(col0) values (2); """
+    qt_sql """ select * from table_ip_default order by col0""";
```
Comment (eldenmoon) [inline regression-test/suites/datatype_p0/ip/test_ip_basic.groovy:169]: also add alter table case

**Review by morrySnow — DISMISSED** [review]

**Review by yiguolei — DISMISSED** [review]

**Comment by amorynan** [comment]
run buildall

**Comment by amorynan** [comment]
run buildall

**Review by eldenmoon — APPROVED** [review]
LGTM

**Review by morrySnow — APPROVED** [review]

**Pull request merged**